charts/loki-stack/Chart.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: "v1"
 name: loki-stack
-version: 2.8.0
+version: 2.8.1
 appVersion: v2.6.1
 kubeVersion: "^1.10.0-0"
 description: "Loki: like Prometheus, but for logs."
 home: https://grafana.com/loki
 icon: https://raw.githubusercontent.com/grafana/loki/master/docs/sources/logo.png
 sources:
-- https://github.com/grafana/loki
+    - https://github.com/grafana/loki
 maintainers:
-- name: Loki Maintainers
-  email: lokiproject@googlegroups.com
+    - name: Loki Maintainers
+      email: lokiproject@googlegroups.com
```

Comment:
Is this change relevant? If not, could you restore it?

Reply:
Not relevant, automation gone wrong.  It's fixed now.